.github/ISSUE_TEMPLATE/release.md

```diff
@@ -18,36 +18,39 @@ Please assign the release manager to the issue.
 
 This issue can be closed when we have completed the following steps (in order).
 Please ensure all artifacts (PRs, workflow runs, Tweets, etc) are linked from
-this issue for posterity. Assuming `vX.Y.0-up.1` is being cut, after upstream
-[crossplane/crossplane][upstream-xp] `vX.Y.0` has been released
+this issue for posterity. Assuming `vX.Y.Z-up.1` is being cut, after upstream
+[crossplane/crossplane][upstream-xp] `vX.Y.Z` has been released
 according to the declared [schedule][uxp-schedule], you should have:
 
-- [ ] Created, or synced if already present, the `release-vX.Y` release branch in [upbound/crossplane][upbound-xp-fork], with upstream [crossplane/crossplane][upstream-xp] release branch, up to the `vX.Y.0` tag, adding any required change specific to the fork, see [here][sync-xp-fork] for more details.
-- [ ] Cut [upbound/crossplane][upbound-xp-fork] `vX.Y.0-up.1` release from the `release-X.Y` branch by:
-  - [ ] Running the [Tag workflow][tag-xp-fork] on the `release-vX.Y` branch with the proper release version, `vX.Y.0-up.1`. Message suggested but not required: `Release vX.Y.0-up.1`.
-  - [ ] Running the [CI workflow][ci-xp-fork] on the `release-vX.Y` branch to build and publish the latest tagged artifacts.
-  - [ ] You should now be able to run: `docker pull xpkg.upbound.io/upbound/crossplane:vX.Y.0-up.1`
+- [ ] Created, or synced if already present, the `release-X.Y` release branch in [upbound/crossplane][upbound-xp-fork], with upstream [crossplane/crossplane][upstream-xp] release branch, up to the `vX.Y.Z` tag, adding any required change specific to the fork, see [here][sync-xp-fork] for more details.
+- [ ] Cut [upbound/crossplane][upbound-xp-fork] `vX.Y.Z-up.1` release from the `release-X.Y` branch by:
+  - [ ] Running the [Tag workflow][tag-xp-fork] on the `release-X.Y` branch with the proper release version, `vX.Y.Z-up.1`. Message suggested but not required: `Release vX.Y.Z-up.1`.
+  - [ ] Running the [CI workflow][ci-xp-fork] on the `release-X.Y` branch to build and publish the latest tagged artifacts.
+  - [ ] You should now be able to run: `docker pull xpkg.upbound.io/upbound/crossplane:vX.Y.Z-up.1`
 - [ ] Created and merged a PR to the `main` branch of [upbound/universal-crossplane][uxp] with the following changes:
-  - [ ] Update any reference to the old latest release to `vX.Y.0-up.1`, such as `CROSSPLANE_TAG` and `CROSSPLANE_COMMIT` in the `Makefile`.
+  - [ ] Update any reference to the old latest release to `vX.Y.Z-up.1`, such as `CROSSPLANE_TAG` and `CROSSPLANE_COMMIT` in the `Makefile`.
```

Comment:
Maybe if we use variable, we could provide `sed` commands to be executed since the change is only done in the `Makefile`?